Remove flush concepts entirely from PendingStateManager

kian-thompson opened the pull request:
# [AB#2496](https://dev.azure.com/fluidframework/internal/_workitems/edit/2496)

[How contribute to this repo](https://github.com/microsoft/FluidFramework/blob/main/CONTRIBUTING.md).

[Guidelines for Pull Requests](https://github.com/microsoft/FluidFramework/wiki/PR-Guidelines#guidelines).

## Description

Previous major version will write new version, so we no longer need to support reading flush messages.

kian-thompson committed:
Remove flush concepts entirely from PendingStateManager

vladsud commented:
> Previous major version will write new version, so we no longer need to support reading flush messages.

I feel like it depends on how quickly we release major versions. As long as it takes longer than 30 days (which is max for session duration we allow, where session includes continuations from serialized state), then we are fine (which is the case today).

Just calling out, given that we are talking about content stored by our users on their machines, in old format.

tylerbutler commented:
@kian-thompson Is there a breaking.md entry related to this change?

tylerbutler committed:
build: Initialize changesets (#14614)

This PR initializes changesets in the client and azure release groups.
This only adds the dependencies and config files. CI enforcement and
tool improvements will be in future PRs.

tylerbutler committed:
Merge branch 'next' into 2496-remove-flush

tylerbutler committed:
Add empty changeset so this change is tracked. Not sure we want a changeset for this ultimately.

kian-thompson committed:
Merge branch 'next' into 2496-remove-flush
